pkg/kubelet/kubelet.go

```diff
@@ -2153,6 +2168,31 @@ func (kl *Kubelet) cleanUpContainersInPod(podID types.UID, exitedContainerID str
 	}
 }
 
+// fastStatusUpdate start a loop that checks the internal node indexer cache for when a CIDR is applied
+// and tries to update pod CIDR immediately. After pod CIDR is updated it fires off a runtime
+// update and a node status update.
+// This should significantly improve latency to ready node by updating pod CIDR, runtime status
+// and node statuses ASAP.
```

Comment:
Please expand the comment to make it clear that this *only* expedites the node status updates when kubelet first starts up. After one successful update, the function returns.

It'd only be good to reflect that in the function name. Maybe `fastStatusUpdateOnce()`?

Reply:
done